src/creep.prototype.js

```diff
@@ -550,10 +564,9 @@ export function moveTo({
     returnData.state = state;
     returnData.path = travelData.path;
   }
-  if (travelData.length === 1 || (range && (range - 1) === rangeToDestination)) {
-    delete creep.memory.moving;
```

Comment:
I was once trying to maintain state in a creep if it was moving or not.  It turned out not to work so I am removing it